src/components/comments/comment.resolver.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  Args,
+  Mutation,
+  Parent,
+  Query,
+  ResolveField,
+  Resolver,
+} from '@nestjs/graphql';
+import {
+  AnonSession,
+  ID,
+  IdArg,
+  ListArg,
+  LoggedInSession,
+  mapSecuredValue,
+  Session,
+} from '../../common';
+import { Loader, LoaderOf } from '../../core';
+import { SecuredUser, UserLoader } from '../user';
+import { CommentThreadLoader } from './comment-thread.loader';
+import { CommentLoader } from './comment.loader';
+import { CommentService } from './comment.service';
+import {
+  Comment,
+  CommentListInput,
+  CommentListOutput,
+  CommentThread,
+  CreateCommentInput,
+  CreateCommentOutput,
+  CreateCommentThreadInput,
+  CreateCommentThreadOutput,
+  DeleteCommentOutput,
+  UpdateCommentInput,
+  UpdateCommentOutput,
+} from './dto';
+
+@Resolver(Comment)
+export class CommentResolver {
```

Comment:
I'm not sure we need all of these queries. I think really the only one that will be used in practice is `Commentable.commentThreads`, and we'll just traverse from there. What do you think?

Reply:
Sounds good to me.